tests/Fixer/Basic/Psr4FixerTest.php

```diff
@@ -47,7 +47,7 @@ class bar {}
 
         $expected = <<<'EOF'
 <?php
-class Psr4_Foo_Bar {}
+class Bar {}
```

Comment:
This is actually correct behaviour - see line 30 in this file, basename is `Bar`, so the class name should equal to it.